### ISSUE TITLE
pie chart renders double doughnut when value is above 200% -- broken in 4.1.6+

### ISSUE THREAD

**Issue by peacechen** [highcharts/highcharts#5361]
We're using the pie chart to render a ring graph with multiple values overlaid on each other. When one of the values is above 200%, one of the rings renders outside of the ring graph.

We've found experimentally that Highcharts 4.1.5 is the latest release that doesn't have this issue. 4.1.6 and above renders the pie chart incorrectly in that scenario. 4.1.10 unfortunately has another bug causing the ring graph to animate with an artifact and doesn't reflect the correct value at the end.

These versions are important because the highcharts versions released on npm include 4.1.0 and 4.2.0 but nothing in between. Thus we're unable to specify 4.1.5 from the npm registry. We had been using the package highcharts-release 4.1.5, but switched to highcharts for CommonJS support.

Would you kindly publish highcharts 4.1.5 on npm? It may need to be a custom tag since there have been newer versions published.

**Comment by jon-a-nygaard** [comment]
@peacechen Hi, could you please provide us with a live demo of the issue for debugging? [JSFiddle](https://jsfiddle.net) works great for this.

@TorsteinHonsi Should we start publishing our minor versions with tags to npm as well?

**Comment by TorsteinHonsi** [comment]
> Should we start publishing our minor versions with tags to npm as well?

We do, but this particular version is probably from before we started publishing to npm. The full tag list for the _highcharts-dist_ repo can be seen at https://github.com/highcharts/highcharts-dist. @jon-a-nygaard can you check out how this corresponds to the available tags on the _highcharts_ npm package?

**Comment by jon-a-nygaard** [comment]
@TorsteinHonsi: Available tags for the highcharts package on npm is the following: 
` ['0.0.1',
  '0.0.2',
  '0.0.3',
  '0.0.4',
  '0.0.5',
  '0.0.6',
  '0.0.8',
  '0.0.9',
  '0.0.10',
  '0.0.11',
  '4.1.10',
  '4.2.0',
  '4.2.1',
  '4.2.2',
  '4.2.3',
  '4.2.4',
  '4.2.5' ]`

Can be found by running `npm view highcharts versions` in a console terminal.

The tags in our github distribution repository is from version 4.0.0 and upwards.

**Comment by TorsteinHonsi** [comment]
Ok. Still waiting for @peacechen to provide a live demo so that we can fix the issue for the next release. Here is what I have tried based on your description, but it renders without error: http://jsfiddle.net/highcharts/0yrnegs6/.

**Comment by peacechen** [comment]
I'll work on the live demo to aid with debugging.

We did point the npm dependency to the Github repo (a specific commit matching 4.1.5), but that unfortunately caused sporadic issues during _npm install_ which I suspect was due to rate limiting.

**Comment by jon-a-nygaard** [comment]
@peacechen Did you point to our [highcharts-dist repository](https://github.com/highcharts/highcharts-dist) or this repository? The highcharts-dist is the one meant for use with npm.

**Comment by peacechen** [comment]
@jon-a-nygaard  When using npm's registry, we pointed to the highcharts package.  For Github we pointed to this repo (not highcharts-dist).  The Github rate limit issue affects any public repo though, so switching to highcharts-dist isn't likely to resolve that.

**Comment by peacechen** [comment]
@TorsteinHonsi Here is a fiddle with the data causing the artifact:
http://jsfiddle.net/0yrnegs6/1/

And don't I feel like a dummy, it's the pie chart not the solidgauge that's doing this.

**Comment by jon-a-nygaard** [comment]
@peacechen It seems to be a duplicate of #5322. It will be fixed in an upcoming maintenance release, and you can find a potential workaround there.

**Comment by peacechen** [comment]
@jon-a-nygaard Thanks for referring me to the originally reported issue.  What is the anticipated release date for the maintenance release with this fix?
